core/cmd/remote_client.go

```diff
@@ -615,6 +615,27 @@ func (cli *Client) deserializeAPIResponse(resp *http.Response, dst interface{},
 	return nil
 }
 
+// parseErrorResponseBody parses response body from web API and returns a single string containing all errors
+func parseErrorResponseBody(responseBody []byte) (string, error) {
+	if responseBody == nil {
+		return "Empty error message", nil
+	}
+
+	var errors models.JSONAPIErrors
+	err := json.Unmarshal(responseBody, &errors)
+	if err != nil {
+		return "", err
+	}
+
+	errorDetails := make([]string, len(errors.Errors))
+
+	for _, errorDetail := range errors.Errors {
+		errorDetails = append(errorDetails, errorDetail.Detail)
+	}
+
+	return strings.Join(errorDetails, "\n"), nil
```

Comment:
maybe an irrelevant micro-optimization, but a `strings.Builder` could be used to build on the fly:
```suggestion
	if err != nil || len(errors.Errors) == 0 {
		return "", err
	}

	var errorDetails strings.Builder
	errorDetails.WriteString(errors.Errors[0])
	for _, errorDetail := range errors.Errors[1:] {
		fmt.Fprintf(errorDetails, "\n%s", errorDetail.Detail)
	}
	return errorDetails.String(), nil
```